.evergreen/config.yml

```diff
@@ -410,32 +410,6 @@ functions:
           export HEAD_SHA=${github_commit}
           bash etc/api_report.sh
 
-  "add PR labels":
-    - command: shell.exec
-      type: test
-      params:
-        shell: "bash"
-        working_dir: src/go.mongodb.org/mongo-driver
-        add_expansions_to_env: true
-        script: |
-          ${PREPARE_SHELL}
-          export CONFIG=$PROJECT_DIRECTORY/.github/labeler.yml
-          export SCRIPT="$DRIVERS_TOOLS/.evergreen/github_app/apply-labels.sh"
```

Comment:
Optional: You can also remove the `apply-labels.sh` script.

Reply:
I wasn't quite sure if those tools were still used by another driver. I'll confirm with @blink1073 and will remove from DET if possible.

Reply:
Ah, I didn't notice it was in a different repo! Yeah, checking with all the drivers is a lot more work. Maybe make a DRIVERS ticket to clean up unused scripts?